a10_octavia/controller/worker/tasks/a10_database_tasks.py

```diff
@@ -1081,6 +1081,15 @@ def execute(self, loadbalancer, role):
                 db_apis.get_session(), vthunder.loadbalancer_id)
             if lb:
                 amphora = self.amphora_repo.get(db_apis.get_session(), load_balancer_id=lb.id)
+                if not amphora:
+                    """
+                        We don't support create loadbalancer in Thunder derice and vthunder in
+                    smae project now. So, this should caused by some error configuration.
```

Comment:
Just a Typo error: 
We don't support create loadbalancer in Thunder **device** and vthunder in
**same** project now. So, this should caused by some error configuration.

Reply:
done